<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>grabDataWidget</class>
 <widget class="QWidget" name="grabDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Carga de Datos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="SizigiagroupBox">
     <property name="title">
      <string>Sizigia: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <layout class="QHBoxLayout" name="sizigiaLayout">
      <property name="topMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Sizigia: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sizigiaComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Enero</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Febrero</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Marzo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Abril</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mayo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Junio</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Julio</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Agosto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Septiembre</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Octubre</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Noviembre</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Diciembre</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Solsticio de Verano</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Solsticio de Invierno</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Equinoccio de Primavera</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Equinoccio de Oto√±o</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="lunaGroupBox">
        <property name="title">
         <string>Luna</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QRadioButton" name="fullMoonRadioButton">
           <property name="text">
            <string>Llena</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="newMoonRadioButton">
           <property name="text">
            <string>Nueva</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="fechaGroupBox">
     <property name="title">
      <string>Fecha:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Fecha / Hora:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dateTimeEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Datos:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="tempMaxLabel">
          <property name="text">
           <string>Temp Max:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="tempMaxDoubleSpinBox">
          <property name="minimum">
           <double>-99990000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="tempMinLabel">
          <property name="text">
           <string>Temp Min:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="tempMinDoubleSpinBox">
          <property name="minimum">
           <double>-990000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="presionLabel">
          <property name="text">
           <string>Precipitacion:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="precipitacionSpinBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="velVientoLabel">
          <property name="text">
           <string>Vel. Viento:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="dirVientoLabel">
          <property name="text">
           <string>Dir. Viento:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="dirVientoSpinBox"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="data1000500Label">
          <property name="text">
           <string>1000/500</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="velVientoDoubleSpinBox">
          <property name="maximum">
           <double>999.990000000000009</double>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="data1000500SpinBox">
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Observaciones:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="observacionesTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="astroInfoButton">
       <property name="text">
        <string>Generar Datos Astrologicos</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="satelmecPushButton">
       <property name="text">
        <string>Cargar Data Archivo</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Cerrar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>grabDataWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>595</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>618</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
